packages/block/test/from-rpc.spec.ts

```diff
@@ -83,9 +83,13 @@ tape('[fromRPC]:', function (t) {
     'should create a block given json data that includes a difficulty parameter of type integer string',
     function (st) {
       const common = new Common({ chain: Chain.Mainnet, hardfork: Hardfork.London })
-      const blockDifficultyAsInteger = blockFromRpc(blockDataDifficultyAsInteger, undefined, {
-        common,
-      })
+      const blockDifficultyAsInteger = blockFromRpc(
+        blockDataDifficultyAsInteger as any,
```

Comment:
Same in this file, guess those `any`s can be removed/changed?

Reply:
Indeed! done